docs/installation/installing-tentacles/polling-tentacles-web-sockets.md

```diff
@@ -0,0 +1,91 @@
+```
+.\Octopus.Server.exe service --instance OctopusServer --stop
+.\Octopus.Server.exe configure --instance OctopusServer --commsListenWebSocket https://+:443/OctopusComms
+.\Octopus.Server.exe service --instance OctopusServer --start
+```
+
+### Testing
+To confirm that the server is successfully configured, open the listen address in your browser. If you are using `+` for the host, replace that with `localhost`. For example `https://localhost:443/OctopusComms`.
+
+If you encounter a certificate warning, ignore it and continue. You should get a page titled `Octopus Server configured successfully`. 
+
+If you do not yet know the thumbprint of your SSL certificate, you can find it by using your browser's developer tools.
+
+If you get your other application or an error, check the [server log file](/reference/log-files.md) to ensure the server is listening on the
+expected prefix (look for `Listener started`)
+
+## Tentacle Setup
+The setup of a WebSocket tentacle is the same as a TCP polling tentacle, except for the thumbprint and the command line option to specify the communications address.
+
+### Registering
+When issuing the `register-with` command during tentacle setup, omit the `--server-comms-port` parameter and specify the `--server-web-socket <address>` parameter. The address to use is the listen prefix (replacing `+` with the hostname) and `https` replaced with `wss` (e.g. `wss://MyServer:443/OctopusComms`). For example:
+```powershell
+.\Tentacle.exe register-with --instance MyInstance --server "https://MyServer/"  --server-web-socket "wss://MyServer:443/OctopusComms" --comms-style TentacleActive --apikey "API-CS0SW5SQJNLUBQCUBPK8LZY3KYO" --environment "Test" --role "Web" 
+```
+
+### Changing an Existing Tentacle
+To change an existing tentacle to poll using WebSockets, run the following commands:
+```powershell
+.\Tentacle.exe service --instance MyInstance --stop
+.\Tentacle.exe configure --reset-trust
+.\Tentacle.exe register-with --instance MyInstance --server "https://MyServer/" --server-web-socket "wss://MyServer:443/OctopusComms" --comms-style TentacleActive --apikey "API-CS0SW5SQJNLUBQCUBPK8LZY3KYO" --environment "Test" --role "Web" 
+.\Tentacle.exe service --instance MyInstance --start
+```
+
+### High Availablity
+When issuing the `poll-server` command to add additional nodes to poll, `--server-comms-port` parameter and specify the `--server-web-socket <address>` parameter. For example:
+
+```powershell
+poll-server --instance MyInstance --server "https://MyServer/"  --server-web-socket "wss://MyServer:443/OctopusComms" --apikey "API-CS0SW5SQJNLUBQCUBPK8LZY3KYO"
+```
+## More Information
+
+### SSL Certificate
+Windows will need to be configured with an SSL certificate on the select address and port. Usually this is done by the other application sharing the port.
+The certificate does _not_ need to be trusted by the server or tentacle machine. The certificate also does not need to match the hostname.
```

Comment:
When you say "trusted", you mean "installed in the certificate store"? Or something else?